### ISSUE TITLE
PDF export has wrong MIME type

### ISSUE THREAD

**Issue by sparmin** [lunarphp/lunar#1976]
- Lunar version: 1.0.0-beta.2
- Laravel Version: 11.23.5
- PHP Version: 8.3
- Database Driver & Version:

### Expected Behaviour:
Exported pdf file should have MIME type "application/pdf".

### Actual Behaviour:
MIME type is "application/octet-stream". This results in validation errors on other web forms.

### Steps To Reproduce:
Go on "Orders". Select an order. Download order as PDF. Check MIME type with online tools.

**Comment by glennjacobs** [comment]
Could you clarify what you mean by...

> This results in validation errors on other web forms.

Thanks

**Comment by sparmin** [comment]
If the file needs to be uploaded on another website, where the form validates the file type with `mime_content_type()`, it results in "application/octet-stream" which can fail the validation.

**Comment by glennjacobs** [comment]
So do you have this automated somehow?

**Comment by sparmin** [comment]
No, I found that by chance. I used the file to test a form of a website I work on and thought that the form broke but instead it uses the said method and fails to validate the file. I haven't tested it with other forms on other sites, but if they do check the MIME type and don't rely only on checking the file extension (which is not a safe way), it will fail there too. In any case the MIME type should match with the file suffix to prevent any possible errors.

**Comment by glennjacobs** [comment]
I believe that mime type is used to force the download of the PDF.

**Comment by sparmin** [comment]
Sorry, I don't understand your point. What are you suggesting?

**Comment by glennjacobs** [comment]
I mean the current set-up using "application/octet-stream" is intended, as we wanted the PDF to download in a browser, not open in the browser.

Happy to have a discussion here to determine if any changes are required.

**Comment by alexmartinfr** [comment]
There is a way to preserve the correct _application/pdf_ MIME Type while still having the browser download the file.

It involves setting the **Content-Disposition** header value to **attachment**.
On the other hand, setting that value to inline will let the browser open the PDF with its integrated viewer, if possible.

Here's how it is managed in _barryvdh/laravel-dompdf_:

```php
    /**
     * Make the PDF downloadable by the user
     */
    public function download(string $filename = 'document.pdf'): Response
    {
        $output = $this->output();
        $fallback = $this->fallbackName($filename);

        return new Response($output, 200, [
            'Content-Type' => 'application/pdf',
            'Content-Disposition' => HeaderUtils::makeDisposition('attachment', $filename, $fallback),
            'Content-Length' => strlen($output),
        ]);
    }

    /**
     * Return a response with the PDF to show in the browser
     */
    public function stream(string $filename = 'document.pdf'): Response
    {
        $output = $this->output();
        $fallback = $this->fallbackName($filename);


        return new Response($output, 200, [
            'Content-Type' => 'application/pdf',
            'Content-Disposition' => HeaderUtils::makeDisposition('inline', $filename, $fallback),
        ]);
    }
```

I tried to look at Lunar's source code for the PDF generation, but can't seem to find where the 'application/octet-stream' Content-Type is defined. If you can point me in the right direction I'd be happy to look at that issue.

**Comment by alecritson** [comment]
Sorry for the delay on this one, so the code which downloads the PDF is in the `Http/Controllers/DownloadPdfController.php` controller. We use https://github.com/barryvdh/laravel-dompdf to generate and stream the PDF.
